src/replacement/moment-timezone/latest.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Timezones are configurable by user. We shouldn't limit available timezones to [DEFAULT_TIMEZONES](https://github.com/allegro/turnilo/blob/master/src/common/models/customization/customization.ts#L28)

Reply:
I see. It could be user configuration-aware I suppose 🤔

Reply:
I've removed this change.